x-pack/plugins/observability_solution/investigate/public/hooks/use_investigation/index.tsx

```diff
@@ -99,10 +99,6 @@ function useInvestigationWithoutContext({
         id: v4(),
         globalWidgetParameters: {
           filters: [],
```

Comment:
We will end up doing a few rounds of clean up anyway. I am just wondering if `filters` are currently used or it is part of `lot of cleanup of unused feature`?

Reply:
I'm not sure where is that one used from, I think we can clean it up later as well